tests/dotnet/UnitTests/ProjectTest.cs

```diff
@@ -4,6 +4,8 @@
 using System.Linq;
 using System.Runtime.InteropServices;
 using System.Text;
+using System.Diagnostics;
+using System.Text.Json;
 
 using Mono.Cecil;
 
```

Comment:
Add nullable enabled please. There are going to be a fair amount of warnings in this code.